compiler/passes/normalize.cpp

```diff
@@ -1274,7 +1282,7 @@ static void normalizeYields(FnSymbol* fn) {
       retval->addFlag(FLAG_YVV);
 
       yield->insertBefore(new DefExpr(retval));
-      insertRetMove(fn, retval, yield);
+      insertRetMove(fn, retval, yield, false);
```

Comment:
Maybe your PR message should say something about not supporting generic array yield types yet?